### PR TITLE
Borrower signs the deed

### PR DESCRIPTION
**Pull request by dawidskainos**

### CODE REVIEW

**Comment by Phemus** [comment]
:+1:

File: app/deed/sign/server.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+def register_routes(blueprint, deed_api):
+    @blueprint.route('/deed/<deed_id>/<borrower_id>/signature',
+                     methods=['POST'])
+    def sign_deed(deed_id, borrower_id):
+        borrower_name = request.form['borrower_name']
+
+        def create_signature():
+            return borrower_name + "_" + strftime("%d/%m/%Y_%H:%M:%S")
```
Comment (endamccormack) [inline app/deed/sign/server.py:12]: when we sign the deed, we should use the iso date time standard. DateTime has a method .isoformat()

Comment (dawidskainos) [inline app/deed/sign/server.py:12]: This signature is a temporary solution, therefore date format doesn't matter at all.

**Comment by endamccormack** [comment]
:+1:

**Pull request merged**